source/model/backtraceStackTrace.ts

```diff
@@ -208,9 +211,7 @@ export class BacktraceStackTrace {
     });
   }
 
-  private isCallingModule(stackFrame: IBacktraceStackFrame): boolean {
-    return (
-      !this.callingModulePath && fs.existsSync(stackFrame.sourceCode) && !stackFrame.sourceCode.includes('node_modules')
-    );
+  private isCallingModule(sourcePath: string): boolean {
+    return !!sourcePath && !this.callingModulePath && !sourcePath.includes('node_modules');
```

Comment:
Did you mean to put this `!!` here? I'm not sure how to interpret this logic

Reply:
Yes - it's expected. 
Let's talk about two use cases:
#1 `sourcePath` is `undefined`/`null`/`empty
#2 `sourcePath` is defined

for #1 
!sourcePath => true 
!!sourcePath => false
 
for #2 
!sourcePath => false
!!sourcePath => true

we want to check by using `!!` if object is defined.

Reply:
Okay sounds good, so `isCallingModule` returns true if:
sourcePath **is** defined, callingModulePath **is not** defined, and sourcePath **is not** a node_module

Reply:
That makes sense and it matches the previous logic too